src/cryptoadvance/specter/api/rest/resource_jwt.py

```diff
@@ -0,0 +1,95 @@
+import jwt
+from flask import current_app as app
+from cryptoadvance.specter.api.rest.base import (
+    BaseResource,
+    rest_resource,
+    AdminResource,
+)
+import uuid
+import datetime
+import logging
+from ...user import *
+from .base import *
+
+from .. import auth
+
+logger = logging.getLogger(__name__)
+
+def generate_jwt(user):
+    payload = {
+        "user": user.username,
+        "exp": datetime.datetime.utcnow() + datetime.timedelta(days=1),
+    }
+    return jwt.encode(payload, app.config["SECRET_KEY"], algorithm="HS256")
+
+def generate_token_id():
+    return str(uuid.uuid4())
+
+@rest_resource
+class ResourceJWT(SecureResource):
```

Comment:
Can you please document all classes, e.g. here is an example:
```suggestion
class ResourceJWT(SecureResource):
    ''' A Resource to manage JWT tokens in order to authenticate against the REST-API
      Other then the other Resources, this endpoint uses BasicAuth to avoid the chicken egg problem
    '''
```